var/spack/repos/builtin/packages/antlr/package.py

```diff
@@ -52,27 +49,19 @@ class Antlr(Package):
     # Unpatched version
     # url='http://dust.ess.uci.edu/nco/antlr-2.7.7.tar.gz')
 
-    variant('cxx', default=False, description='Enable ANTLR for C++')
-    variant('java', default=False, description='Enable ANTLR for Java')
+    variant('cxx',    default=True,  description='Enable ANTLR for C++')
```

Comment:
I don't see why this shouldn't default to True. A C++ compiler is almost always available.